packages/components/src/Hero/HomepageHero.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from "react";
+import { Button, buttonSizes } from "../Button";
+import { HeroLabel, HeroHeading, HeroBlockTextLink, HeroSmallText } from "./styledComponents";
+
+export class HomepageHero extends React.Component {
+  public render(): JSX.Element {
+    return (
+      <>
+        <HeroLabel>Civil Registry</HeroLabel>
+        <HeroHeading>
+          The Civil Registry is a whitelist of community-approved newsrooms that have publishing rights on Civil.
```

Comment:
can we move this text to a constants file (which we need to create and start using for all text)